### ISSUE TITLE
undefined is not an object (evaluating '_this.refs.input.setNativeProps')

### ISSUE THREAD

**Issue by hnguyen14** [nulrich/RCTAutoComplete#26]
this error is triggered when the input field already has value and I'm trying to modify it.

The errors from from this code block (RCTAutoComplete.ios.js)

```
if (text !== this.props.value && typeof this.props.value === 'string') {
  this.refs.input.setNativeProps({
    text: this.props.value,
  });
}
```

but there doesn't seem to be a ref="input" anywhere in the component. Am I missing something ?

**Comment by hnguyen14** [comment]
Hi ... this bug has been fixed with https://github.com/nulrich/RCTAutoComplete/pull/31 ... Can you publish this change to NPM ? I think this is a critical bug that prevent user to re-enter value for the input field (throws the error mentioned in the title).

**Comment by nulrich** [comment]
I'm fixing #29 and I will do the christmas release

**Comment by nulrich** [comment]
Fixed by #31 thanks
